stretch_description/launch/display.launch.py

```diff
@@ -0,0 +1,34 @@
+from ament_index_python.packages import get_package_share_path
+
+from launch import LaunchDescription
+from launch.substitutions import Command
+
+from launch_ros.actions import Node
+
+
+def generate_launch_description():
+    stretch_description_path = get_package_share_path('stretch_description')
+    robot_description_content = Command(['xacro ', str(stretch_description_path / 'urdf' / 'stretch_main.xacro')])
+
+    joint_state_publisher_gui_node = Node(
+        package='joint_state_publisher_gui',
+        executable='joint_state_publisher_gui'
```

Comment:
The joints starting at zeros visualizes the robot intersecting with itself. We can use the 'zeros' parameter to change the stow config:

```python
joint_state_publisher_gui_node = Node(
    package='joint_state_publisher_gui',
    executable='joint_state_publisher_gui',
    parameters=[{'zeros.joint_lift': 0.2, 'zeros.joint_wrist_yaw': 3.4}]
)
```